### PR TITLE
VZ-10263 Add advisory to use Grafana database if replicas > 1

### PR DESCRIPTION
**Pull request by desagar**

### CODE REVIEW

**Review by anders-swanson — APPROVED** [review]

File: content/en/docs/observability/monitoring/configure/grafana/grafana.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ aliases:
 ---
 
 ### Configure a Grafana database
-
+{{< alert title="NOTE" color="primary" >}}You must configure a database for Grafana if you set Grafana `replicas` to more than 1.
```
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/configure/grafana/grafana.md:11]: In general, you should **not** place a NOTE as the leading sentence in a document. I suggest putting it after these sentences, "By default, Verrazzano automatically installs and configures a Grafana database. However, you can use your own external database."

-> If you set Grafana `replicas` to a value greater than 1, then you must configure a database for Grafana. (if.., then construct)

**Review by rosemarymarano — COMMENTED** [review]
Edits/suggestions.

File: content/en/docs/observability/monitoring/configure/grafana/grafana.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ aliases:
...
+{{< /alert >}}
 By default, Verrazzano automatically installs and configures a Grafana database. However, you can use your own external database.
 
 If you prefer to use your own Grafana database, complete the following steps:
```
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/configure/grafana/grafana.md:15]: After the Note text, you might consider changing this stem sentence as follows, "To use your own Grafana database, complete the following steps:"

**Review by rosemarymarano — COMMENTED** [review]
One more suggestion

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**